mrmustard/training/parametrized.py

```diff
@@ -42,8 +47,8 @@ class Parametrized:
 
     def __init__(self, **kwargs):  # NOTE: only kwargs so that we can use the arg names
         owner = f"{self.__class__.__qualname__}"
-        self.param_names = {}
-        for k, (name, value) in enumerate(kwargs.items()):
+        self.param_names = []  # list of parameter names to preserve order
```

Comment:
Can get them directly from the dict instead of appending
```suggestion
        self.param_names = tuple(kwargs.keys())  # list of parameter names to preserve order
```

Reply:
but that includes kw that end in _trainable and _bounds no?